src/node/fetch/fetch.node.test.ts

```diff
@@ -15,225 +15,292 @@ const timers = new MockTimers();
+        const fetchInstance = new NodeFetch(timers);
 
-      result = await fetchInstance.fetch(`${mockServer.url}/test`, {
-        method: 'GET',
+        result = await fetchInstance.fetch(`${mockServer.url}/test`, {
+          method: 'GET',
+        });
       });
-    });
 
-    it('should call json', async () => {
-      expect(responseJsonMock).toBeCalled();
-    });
+      it('should call json', async () => {
+        expect(responseJsonMock).toBeCalled();
+      });
 
-    it('should return json object in body', async () => {
-      expect(result.body).toBeInstanceOf(Object);
-      expect(result.body).toStrictEqual({
-        foo: 'bar',
+      it('should return json object in body', async () => {
+        expect(result.body).toBeInstanceOf(Object);
+        expect(result.body).toStrictEqual({
+          foo: 'bar',
+        });
       });
     });
-  });
 
-  describe('when text/plain content type received', () => {
-    let responseTextMock: jest.Mock;
-    let result: any;
+    describe('when text/plain content type received', () => {
+      let responseTextMock: jest.Mock;
+      let result: any;
 
-    beforeEach(async () => {
-      responseTextMock = jest.fn().mockResolvedValue('foobar');
+      beforeEach(async () => {
+        responseTextMock = jest.fn().mockResolvedValue('foobar');
 
-      mocked(fetch).mockResolvedValue({
-        headers: {
-          forEach: jest.fn((callbackfn: ForEachCallbackFunction) => {
-            callbackfn('text/plain', 'content-type');
-          }),
-        },
-        text: responseTextMock,
-      } as any);
+        mocked(fetch).mockResolvedValue({
+          headers: {
+            forEach: jest.fn((callbackfn: ForEachCallbackFunction) => {
+              callbackfn('text/plain', 'content-type');
+            }),
+          },
+          text: responseTextMock,
+        } as any);
 
 
-      const fetchInstance = new NodeFetch(timers);
+        const fetchInstance = new NodeFetch(timers);
 
-      result = await fetchInstance.fetch(`${mockServer.url}/test`, {
-        method: 'GET',
+        result = await fetchInstance.fetch(`${mockServer.url}/test`, {
+          method: 'GET',
+        });
+      });
+
+      it('should call text', async () => {
+        expect(responseTextMock).toBeCalled();
       });
-    });
 
-    it('should call text', async () => {
-      expect(responseTextMock).toBeCalled();
+      it('should return plain text in body', async () => {
+        expect(result.body).toBe('foobar');
+      });
     });
 
-    it('should return plain text in body', async () => {
-      expect(result.body).toBe('foobar');
+    describe('when binary content type received', () => {
+      const binaryContentTypes = [
+        'application/octet-stream',
+        'audio/mp3',
+        'audio/wav',
+        'audio/wav;rate=8000',
+        'video/mp4',
+        'image/jpeg',
+      ];
+
+      for (const contentType of binaryContentTypes) {
```

Comment:
question: any idea why this doesn't use https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Don't know. But I just rewrote when I changed `content-type` handling in https://github.com/superfaceai/one-sdk-js/blob/dev/src/core/interpreter/http/filters.ts#L259

Reply:
Now (when I ran the test) I know why. `beforeEach` can't be used inside `test.each`.